plugins/quetz_transmutation/quetz_transmutation/api.py

```diff
@@ -12,6 +12,8 @@
 from quetz.jobs.models import Job
 from quetz.pkgstores import PackageStore
 
+from .rest_models import PackageSpec
```

Comment:
@btel I think this file is missing?

Reply:
ah, sorry, I was tired. but why did the tests pass 🤔 ?!